sunkit_image/flct/__init__.py

```diff
@@ -0,0 +1 @@
+from .flct import *
```

Comment:
Add an import from utils here as well. 

The aim of the utils file was to split up the code internally but from an importing point of view, we want the use to just import flct and have access to them.